collector/collector.go

```diff
@@ -72,16 +66,25 @@ func (d *typedDesc) mustNewConstMetric(value float64, labels ...string) promethe
 	return prometheus.MustNewConstMetric(d.desc, d.valueType, value, labels...)
 }
 
-func NewExporter(address string, logger log.Logger) Exporter {
+type ChronyCollectorConfig struct {
+	Address string
+	Timeout time.Duration
+
+	CollectSource      bool
```

Comment:
Minor nit, naming mismatch.

```suggestion
	CollectSources     bool
```